tv-shows/src/components/shared/stars/StarsRatingInput.tsx

```diff
@@ -1,23 +1,41 @@
-import { Flex } from "@chakra-ui/react";
-import { StarIcon } from "./StarIcon";
+import { Flex, FormControl, FormErrorMessage } from "@chakra-ui/react";
 import { StarInput } from "./StarInput";
 
-export const StarsRatingInput = () => {
+interface StarsRatingInputProps {
+  register: any;
+  errors: any;
+  required?: boolean;
+}
+
+export const StarsRatingInput = ({
+  register,
+  errors,
+  required = false,
+}: StarsRatingInputProps) => {
   return (
-    <Flex className="stars-rating">
-      <input
-        defaultChecked
-        className="rating-input rating-input--none"
-        name="rating"
-        id="rating-none"
-        value="0"
-        type="radio"
-      />
-      <StarInput id="1" />
-      <StarInput id="2" />
-      <StarInput id="3" />
-      <StarInput id="4" />
-      <StarInput id="5" />
-    </Flex>
+    <FormControl isInvalid={required && !!errors.rating}>
+      <Flex className="stars-rating">
+        <input
+          defaultChecked
+          className="rating-input rating-input--none"
+          {...register("rating", {
+            required: required ? "Rating is required" : false,
+            validate: (value: string) =>
+              parseInt(value) > 0 || "Rating must be greater than 0",
+          })}
+          id="rating-none"
+          value="0"
+          type="radio"
+        />
+        <StarInput id="1" register={register} />
```

Comment:
Za sad je ovo okejish, možemo ovo i elegantnije riješiti.
Prođemo to nekad u sljedećim predavanjima (zadaćama)